app/pages/admin/project-status.jsx

```diff
@@ -124,17 +124,23 @@ class ProjectStatus extends Component {
       .catch(error => this.setState({ error }));
   }
 
+  saveProject(project) {
+    return project.save()
+      .catch(error => this.setState({ error }));
+  }
+
   handleFeaturedProjectChange({ target }) {
-    const { featured, project } = this.state;
-    if (featured) {
-      featured.update({ featured: false });
-      featured.save()
-        .then(() => this.setState({ featured }))
+    const { featuredProject, project } = this.state;
+    project.update({ featured: target.checked });
+    if (featuredProject) {
+      featuredProject.update({ featured: false });
+      return this.saveProject(featuredProject)
+        .then(() => this.saveProject(project))
+        .then(() => this.setState({ featuredProject: project }))
```

Comment:
`.save()` resolves with the updated resource from Panoptes so `.save().then( project => ...)`